src/renderers/SymbolInstanceRenderer.js

```diff
@@ -36,7 +36,7 @@ const extractOverridesHelp = (subLayer: any, output: any) => {
     return;
   }
 
-  if (subLayer._class === 'group' && subLayer.layers) {
```

Comment:
This can be removed because I now handle checking for layers within the `findInGroup` method.